libs/common/src/services/organization/organization.service.ts

```diff
@@ -26,6 +26,10 @@ export class OrganizationService implements InternalOrganizationServiceAbstracti
       .subscribe();
   }
 
+  get$(id: string): Observable<Organization> {
```

Comment:
suggestion (non-blocking): This can (optionally) be future proofed

```suggestion
  get$(id: string): Observable<Organization | undefined> {
```